skyportal/handlers/api/observation_plan.py

```diff
@@ -42,6 +44,95 @@
 TREASUREMAP_URL = cfg['app.treasuremap_endpoint']
 
 
+def post_observation_plan(plan, user_id, session):
+    """Post ObservationPlan to database.
+    plan: dict
+        Observation plan dictionary
+    user_id : int
+        SkyPortal ID of User posting the GcnEvent
+    session: sqlalchemy.Session
+        Database session for this transaction
+    """
+
+    user = session.query(User).get(user_id)
```

Comment:
What prevents me from using someone else's user_id and posting stuff? Is that enforced in the handler that is calling this?

Reply:
Perhaps it is better to pass in a User object rather than an ID, that way the thing passing in the object will be more inclined to not allow a user to input the ID themselves without checking that the user is legal?

Reply:
@guynir42 This was to simplify the use from, for example microservices, which Stefan had suggested previously. As you intuited, this is to just be called from within handlers, and I would think making a new user in the session is cleaner.